packages/frontend/src/components/navigation/UserAccount.js

```diff
@@ -2,8 +2,10 @@ import React from 'react';
 import styled from 'styled-components';
 
 import classNames from '../../utils/classNames';
+import ClickToCopy from '../common/ClickToCopy';
 import ChevronIcon from '../svg/ChevronIcon';
 import UserIcon from '../svg/UserIcon';
+import CopyIcon from '../svg/CopyIcon';
```

Comment:
```suggestion
import CopyIcon from '../svg/CopyIcon';
import UserIcon from '../svg/UserIcon';
```
We have an eslint rule which sorts imports alphabetically, is your eslint setup correctly? 😄

Reply:
Sorry, will take a look at this shortly